src/main/java/cceclipseplugin/core/CCIgnore.java

```diff
@@ -141,7 +151,7 @@ private void serverCleanUp(IProject p) {
 			for (String ignored : ignoredFiles) {
 				if (path.startsWith(ignored) || path.equals(ignored)) {
 					// send delete request for fileID
-					System.out.println(String.format("Cleaning up %s from server", path));
+					Logger.getInstance().log(IStatus.INFO, String.format("Cleaning up %s from server", path));
```

Comment:
Should this be using the `logger` variable created above? It appears that `logger` is initialized but not ever used since this line uses the static function `Logger.getInstance()`.